exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -161,14 +162,15 @@ def send_bytes(bytes, timeout:) # rubocop:disable Metrics/CyclomaticComplexity,
             when Net::HTTPServiceUnavailable, Net::HTTPTooManyRequests
               response.body # Read and discard body
               redo if backoff?(retry_after: response['Retry-After'], retry_count: retry_count += 1, reason: response.code)
+              log_request_failure(response.code)
               FAILURE
             when Net::HTTPRequestTimeOut, Net::HTTPGatewayTimeOut, Net::HTTPBadGateway
               response.body # Read and discard body
               redo if backoff?(retry_count: retry_count += 1, reason: response.code)
+              log_request_failure(response.code)
```

Comment:
These two calls will double count if `backoff?` returns `false` because `backoff?` also does `@metrics_reporter.add_to_counter('otel.otlp_exporter.failure' ...)`.

Reply:
Nice catch 😅  - will 🤔  and 💻

Reply:
86f99e80fa1ecbb4cddfec6ffdfad1a8aa52cf5b